docker/kanister-elasticsearch/image/Dockerfile

```
@@ -10,4 +10,3 @@ ADD docker/kanister-elasticsearch/image/esdump-setup.sh /esdump-setup.sh
 RUN chmod +x /esdump-setup.sh && sync && /esdump-setup.sh
 
 CMD [ "/usr/bin/tail", "-f", "/dev/null" ]
```

We should keep the new line at the end right?
```suggestion
CMD [ "/usr/bin/tail", "-f", "/dev/null" ]

```

---

we had an extra extra line Ankit, I will mark this as resolved.